functions/activate-batch-auto-background.ts

```diff
@@ -172,17 +169,17 @@ export async function handle(_?: any, db?: DynamoDB, contract?: any) {
   })
 }
 
-export const handler = async (event) => {
-  const json = JSON.parse(event.body)
-  if (json.password !== process.env.PASSWORD) {
-    console.log('Unauthorized access')
-    return {
-      statusCode: 401
-    }
-  }
+export const handler = async () => {
+  // const json = JSON.parse(event.body)
+  // if (json.password !== process.env.PASSWORD) {
+  //   console.log('Unauthorized access')
+  //   return {
+  //     statusCode: 401
+  //   }
+  // }
```

Comment:
We absolutely cannot remove this section as it will get uploaded to netlify's admin backend and anyone can call it from outside. Need to keep the password protection there.